testcases/OpTestEnergyScale.py

```diff
@@ -58,7 +58,7 @@ def setUp(self):
     #
     def run_ipmi_cmd(self, i_cmd):
         l_cmd = i_cmd
-        l_res = self.cv_IPMI.ipmitool_execute_command(l_cmd)
+        l_res = self.cv_IPMI.ipmitool_run(l_cmd)
```

Comment:
======================================================================
ERROR: runTest (testcases.OpTestEnergyScale.OpTestEnergyScaleDCMIstandby)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestEnergyScale.py", line 240, in runTest
    self.run_ipmi_cmd(BMC_CONST.IPMI_DCMI_DISCOVER)
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestEnergyScale.py", line 62, in run_ipmi_cmd
    l_res = self.cv_IPMI.ipmitool_run(l_cmd)
AttributeError: OpTestIPMI instance has no attribute 'ipmitool_run'

----------------------------------------------------------------------
Ran 2 tests in 60.498s

FAILED (errors=2)


Still there is a typo